DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -20,7 +20,7 @@ namespace ticl {
   public:
     typedef math::XYZVector Vector;
 
-    enum IterationIndex { TRKEM = 0, EM, TRKHAD, HAD, MIP };
+    enum IterationIndex { TRKEM = 0, EM, TRKHAD, HAD, MIP, SIM, SIM_CP };
```

Comment:
can you comment if and how this change to the dataformat affects backwards compatibility?

Reply:
It should not, IMHO: it does not change the meaning of the old `labels`.
It will add additional `labels` that, by construction, will not be present in old samples.
The only issue is reading new samples with an old release, but that is `forward compatibility`, which is not supported by CMSSW, right?